enterprise/com/src/main/java/org/neo4j/com/storecopy/TransactionCommittingResponseUnpacker.java

```diff
@@ -219,12 +222,6 @@ private void applyQueuedTransactions() throws IOException
                     }
                 }
             }
-            catch ( IOException e )
-            {
-                // Kernel panic is done on this level, i.e. append and apply doesn't do that themselves.
```

Comment:
This panic might still be needed, because what if we get an OOM when we try to close the transactions (after appending)?